File: C++/Iniciante/1041.cpp
#include<iostream>
#include<iomanip>
#include<cmath>

using namespace std;

int main (){
    
    double x, y;
    cin >> fixed >> setprecision(1);
    cin >> x >> y;
    if((x==0) && (y==0)){
          cout << "Origem" << endl;
    }else if((x>0) && (y>0)){
          cout << "Q1" << endl;
    }else if((x<0) && (y>0)){
          cout << "Q2" << endl;
    }else if((x<0) && (y<0)){
          cout << "Q3" << endl;
    }else if((x>0) && (y<0)){
          cout << "Q4" << endl;
    }else if(x==0){
          cout << "Eixo Y" << endl;
    }else if(y==0){
          cout << "Eixo X" << endl;
    }    
    
system("pause");    
return 0;    
}

File: Python3/Iniciante/1041.py
# -*- encoding: utf-8 -*-

N = input()

X = float(N.split()[0])
Y = float(N.split()[1])

if (X==0 and Y==0):
    print("Origem")
elif (X==0):
    print("Eixo Y")
elif (Y==0):
    print("Eixo X")
elif (X>0 and Y>0):
    print("Q1")
elif (X<0 and Y>0):
    print("Q2")
elif (X<0 and Y<0):
    print("Q3")
else:
    print("Q4")
